indico/modules/events/templates/footer.html

```diff
@@ -48,13 +30,18 @@ <h4>{% trans %}Calendaring{% endtrans %}</h4>
                     <img src="{{ url_for('assets.image', filename='google_calendar.gif') }}" alt="" border="0">
                 </a>
             </div>
+            <div class="social-site">
+                <a href="https://outlook.office.com/calendar/deeplink/compose?{{ outlook_calendar_params|urlencode }}&path=%2Fcalendar%2Faction%2Fcompose&rru=addevent" target="_blank">
```

Comment:
What about adding `&path=%2Fcalendar%2Faction%2Fcompose&rru=addevent` to the `outlook_calendar_params` to have everything in one place?